visual-js/visual-snapshots/src/version.ts

```diff
@@ -0,0 +1 @@
+export const clientVersion = "PKG_VERSION";
```

Comment:
Please update the path in `visual-js\replace_pkg_version.sh` so that the version is picked up

Reply:
Oh, wasn't aware of that! Updated